spring-webmvc/src/test/java/org/springframework/web/servlet/mvc/method/annotation/CrossOriginTests.java

```diff
@@ -345,6 +376,34 @@ public void baz() {
 
 	}
 
+	@Target({ElementType.METHOD, ElementType.TYPE})
+	@Retention(RetentionPolicy.RUNTIME)
+	@Documented
+	@CrossOrigin
+	private @interface CrossOriginMetaAnnotation {
```

Comment:
Since I'm handling the JIRA issue now, I'll fix it on my own when I merge the commit.

cheers!